test/metabase/test/util.clj

```diff
@@ -224,10 +224,11 @@
 
 (u/strict-extend (class User)
   test/WithTempDefaults
-  {:with-temp-defaults (fn [_] {:first_name (random-name)
-                                :last_name  (random-name)
-                                :email      (random-email)
-                                :password   (random-name)})})
+  {:with-temp-defaults (fn [_] {:first_name       (random-name)
+                                :last_name        (random-name)
+                                :email            (random-email)
+                                :password         (random-name)
+                                :login_attributes nil})})
```

Comment:
What exactly are you trying to accomplish by adding `login_attributes` here? It would be `NULL` either way so this isn't doing anything. 😖

Reply:
I was getting very odd failures when I tried to implement the keyword args to that `invite-user!` function. The issue ended up being a defrecord code reloading issue. I added this because I thought the issue was here, I'll remove it.

Reply:
in the REPL? Or in tests

Reply:
REPL, if I compiled and loaded `metabase.models.user` more than once, I would weird failures that I could only get to go away after restarting the repl.

Reply:
Yeah because the class changes. You just need to do

```clj
(require 'metabase.test.util :reload)
```

Reply:
:+1: that works and is much better